bookwyrm/templates/snippets/create_status_form.html

```diff
@@ -31,18 +33,34 @@
         {% if type == 'quotation' %}
         <textarea name="quote" class="textarea" id="id_quote_{{ book.id }}_{{ type }}" placeholder="{{ placeholder }}" required></textarea>
         {% else %}
-        <textarea name="content" class="textarea" id="id_content_{{ book.id }}_{{ type }}" placeholder="{{ placeholder }}" required></textarea>
+        {% include 'snippets/content_warning_field.html' with parent_status=status %}
+        <textarea name="content" class="textarea" id="id_content_{{ book.id }}{{reply_parent.id}}{{ uuid }}_{{ type }}" placeholder="{{ placeholder }}" {% if type == 'reply' %} aria-label="Reply"{% endif %} required></textarea>
         {% endif %}
     </div>
     {% if type == 'quotation' %}
     <div class="control">
         <label class="label" for="id_content_{{ book.id }}_quote">Comment:</label>
+        {% include 'snippets/content_warning_field.html' with parent_status=status %}
         <textarea name="content" class="textarea is-small" id="id_content_{{ book.id }}_quote"></textarea>
     </div>
     {% endif %}
-    <div class="control is-grouped">
-        {% include 'snippets/privacy_select.html' %}
-        <button class="button is-primary" type="submit">post {{ type }}</button>
+    <input type="checkbox" class="hidden" name="sensitive" id="show-spoilers-{{ uuid }}" {% if status.content_warning %}checked{% endif %}>
```

Comment:
I think that since this button is fully hidden (with css and aria-hidden) and isn't needed in the ui with a screenreader or visually, this is actually okay. not an ideal solution to have a secret hidden checkbox but that's an improvement for another day